packages/fxa-settings/src/components/LinkRememberPassword/index.tsx

```diff
@@ -25,14 +29,27 @@ const LinkRememberPassword = ({ email }: LinkRememberPasswordProps) => {
     linkHref = params.size > 0 ? `/?${params}` : '/';
   }
 
+  const recordGleanMetric = () => {
+    if (gleanMetric === 'passwordResetEmailConfirmation') {
+      GleanMetrics.passwordReset.emailConfirmationSignin();
+    }
```

Comment:
Interesting.  We can't call the glean function from wherever `gleanMetric` is passed in?

Reply:
Oh I guess the click handler is in here, and we don't accept additional click handling code.

Reply:
That's a better idea. I've modified the component to take in optional/additional click handling. Thanks!